include/mrdox/Metadata/Function.hpp

```diff
@@ -71,12 +71,39 @@ union FnFlags1
     BitField<6, 7, FunctionKind> functionKind;
 };
 
+// KRYSTIAN TODO: attributes (nodiscard, deprecated, and carries_dependency)
+// KRYSTIAN TODO: flag to indicate whether this is a function parameter pack
+/** Represents a single function parameter */
+struct Param
+{
+    /** The type of this parameter */
+    TypeInfo Type;
+
+    /** The name of this parameter, if any */
```

Comment:
/** The parameter name.

    Unnamed parameters are represented by empty strings.
*/